core/pom.xml

```diff
@@ -76,4 +79,13 @@
       </build>
     </profile>
   </profiles>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.io</groupId>
+      <artifactId>shared</artifactId>
+      <version>0.1.0-SNAPSHOT</version>
```

Comment:
I made a change to show that is the current version for the library ${version}. Does that explain?